bin/eva-sub-cli.py

```diff
@@ -58,8 +53,6 @@ def get_version():
     vcf_group.add_argument('--vcf_files', nargs='+', help="One or several vcf files to validate")
     vcf_group.add_argument('--assembly_fasta',
                            help="The fasta file containing the reference genome from which the variants were derived")
-    vcf_group.add_argument("--vcf_files_mapping",
-                           help="csv file with the mappings for vcf files, fasta and assembly report")
```

Comment:
Can you update the readme as well? It's just easier to keep the documentation up-to-date when we do it in parallel with the code updates.

Reply:
updated